build-release.sh

```diff
@@ -392,6 +372,14 @@ if [ "${build_classical}" == "1" ]; then
   #cd ..
   #rm -rf visionos_xcode
 
+  ## .NET bindings ##
+
+  dotnetname="godot-dotnet-${templates_version}"
+  mkdir ${dotnetname}
+  cp out/dotnet/* ${dotnetname}/
+  zip -q -9 -r "${reldir}/${dotnetname}.zip" ${dotnetname}
+  rm -rf ${dotnetname}
```

Comment:
Any further steps we can take to make this easier to use?
E.g. provide a Python script that configures a local Nuget source and installs those packages.

Reply:
We can provide a script, but whatever directory we use can't be removed or it could break restoring packages in any C# project (including non-Godot projects). Also, users would probably want to remove the local NuGet source after the previews and I expect them to forget.

So I think automating this with a script may cause more trouble than the problems it fixes, I'm getting flashbacks to GodotNuGetFallbackFolder.